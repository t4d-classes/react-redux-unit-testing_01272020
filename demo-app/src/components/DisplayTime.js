import React from 'react';

export const DisplayTime = ({ timeString }) => {

  return <div>{timeString}</div>;

};