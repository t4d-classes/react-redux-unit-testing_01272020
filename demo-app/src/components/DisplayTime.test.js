import React from 'react';
import { render } from 'enzyme';

import { DisplayTime } from './DisplayTime';

describe('DisplayTime Snapshot Tests', () => {

  test('DisplayTime displays Hello World', () => {

    const displayTimeHtml = render(
      <DisplayTime timeString="Mon, 27 Jan 2020 12:18:41 GMT" />
    ).html();

    const component = JSON.stringify(displayTimeHtml);

    expect(component).toMatchSnapshot();

  });

});