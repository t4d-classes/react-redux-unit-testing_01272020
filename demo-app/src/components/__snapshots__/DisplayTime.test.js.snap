// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayTime Snapshot Tests DisplayTime displays Hello World 1`] = `"\\"Mon, 27 Jan 2020 12:18:41 GMT\\""`;
